entrypoint/versionbump/canal_test.go

```diff
@@ -32,22 +32,22 @@ var _ = Describe("VersionTemplate Upgrade:", func() {
 	})
 
 	It("Verifies bump version on rke2 for canal with calico and flannel versions", func() {
```

Comment:
Suggest to have separate tests for each component type. `Canal with Calico` and `Flannel with Calico`

Reply:
sure , can we add this as a separate ticket ? since i am not changing any logic here  on this PR. 

and if i do that now i will need further testing not related.

Reply:
Sure that can be separate ticket

Reply:
https://github.com/rancher/qa-tasks/issues/1026